src/store/aggregator/aggregator.slice.ts

```diff
@@ -121,6 +152,10 @@ const aggregatorSlice = createSlice({
       state.startingTokenBalance.state = 'idle';
       state.startingTokenBalance.data = undefined;
       state.txDetails = undefined;
+      state.incentives.state = 'idle';
+      state.incentives.data = {};
+      state.receiveAmount = undefined;
+      state.slippageSlider = 2.5;
```

Comment:
Does not match spec, discuss with @john-light.

Reply:
We decided to remove the slippage tolerance setting for now and use a sane default that is applied in the background. Spec on Jira has been updated to reflect this.